### PR TITLE
[NEW] RocketChatLinkButton Component

### PR DESCRIPTION
**Pull request by sidmohanty11**
Created a new RocketChatLinkButton Component which by default redirects to https://open.rocket.chat

https://user-images.githubusercontent.com/73601258/150687434-46c98142-5c44-4586-96eb-11d6c9870cf0.mp4

### CODE REVIEW

File: app/components/menubar.js [hunk 1]
```diff
@@ -1,23 +1,24 @@
-import { useState } from 'react';
-import { Navbar, Nav, NavDropdown, Container, Col, Row } from 'react-bootstrap';
-import styles from '../styles/Menubar.module.css';
+import { useState } from "react";
```
Comment (Sing-Li) [inline app/components/menubar.js:1]: Menubar is NOT your component. Keep your own component completely independent.

File: app/styles/Menubar.module.css [hunk 2]
```diff
@@ -18,7 +18,7 @@
   background-color: #ff5050;
   color: transparent;
   width: 0%;
-  content: '.';
+  content: ".";
```
Comment (Sing-Li) [inline app/styles/Menubar.module.css:21]: Menubar is *not* your component.   Keep your component independent.

Any coupling should be done via the bootstrap "design language"

**Review by Sing-Li — CHANGES_REQUESTED** [review]

**Comment by Sing-Li** [comment]
Please revise this PR.   Do not keep closing and openning new PRs.  Thanks.

**Comment by sidmohanty11** [comment]
@Sing-Li I've added the rocketchatlinkbutton component and kept it independent, please let me know if we need to make some other changes. 

Thank you

**Comment by Sing-Li** [comment]
@sidmohanty11    Please resolve the trivial conflict and  it will be good-to-merge.  Thx

**Comment by sidmohanty11** [comment]
Done! @Sing-Li

**Pull request merged**